dev/TabView/TabView.xaml

```diff
@@ -60,6 +60,8 @@
                                 Grid.Column="2"
                                 x:Name="AddButton"
                                 Content="&#xE710;"
+                                VerticalAlignment="Stretch"
```

Comment:
Does this cause the Button to take all the vertical space and look taller when you hover over it? Can you take a screenshot of the hovering state? @stmoy @yulikl FYI

Reply:
The VerticalAlignment="Stretch" is needed to prevent the "Add" button to be stuck to the bottom when text-scaling is increased:
![image](https://user-images.githubusercontent.com/16122379/69477376-e5a9be00-0de5-11ea-816d-269511d1f8d7.png)


Current hover look:
![image](https://user-images.githubusercontent.com/16122379/69477355-8055cd00-0de5-11ea-8d43-ab9ed52bf888.png)

Reply:
Got it. OK, thanks!